.gitignore

```diff
@@ -87,7 +87,7 @@ nekoyume/compose/results
 # Genesis Block
 /nekoyume/Assets/StreamingAssets/genesis-block*
 /nekoyume/Assets/StreamingAssets/genesis-block-dev*
-!/nekoyume/Assets/StreamingAssets/genesis-block-9c-main
```

Comment:
Is this still needed?

Reply:
Maybe?

Reply:
Hm.. I'll change the genesis block path to /planets/0x000000000000/genesis if it is genesis-block-9c-main.